docs/contributor/04-30-ci-cd.md

```diff
@@ -70,32 +67,23 @@ The following CI jobs run on the main branch.
+| [`Main Integration / Wait for image build`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L20)                                    | Waits for the operator image build to finish so that the image-dependent jobs can be executed.                                                                             |
+| [`post-api-gateway-manager-build`](https://github.com/kyma-project/test-infra/blob/6fa2829ff262d300663335f80f67050d21b1ae5f/prow/jobs/kyma-project/api-gateway/api-gateway-manager-build.yaml#L59)                                | Builds Kyma API Gateway Operator image and pushes it to the `prod` registry.                                                                                               |
+| [`Main Integration / Integration tests`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L44)                                       | Executes the main integration test suite for the API Gateway module on a k3d cluster. The job uses Istio module.                                    |
+| [`Main Integration / Upgrade tests`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L59)                                           | Executes the upgrade integration test suite for the API Gateway module and verifies if the existing release can be successfully upgraded to the new version.               |
+| [`Main Integration / Custom domain integration GCP`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L75)                           | Executes the integration test suite that verifies the functional correctness of Kyma API Gateway Operator. The test suite uses a custom domain and a Gardener GCP cluster. |
+| [`Main Integration / Custom domain integration AWS`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L94) | Executes the integration test suite that verifies the functional correctness of the API Gateway Operator. The test suite uses a custom domain and a Gardener AWS cluster.  |
+| [`Main Integration / Slack Notification`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/main-integration.yaml#L113)                                      | Sends a Slack notification to the team's channel if any pipelines fail on the main branch.                                                                                 |
 
 ## Pipelines Running on the Release Branch
 
 The following CI jobs generate release artifacts. They are triggered on Git tag creation.
```

Comment:
Do we need to have the [release workflow](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/create-release.yaml) also documented? Since this is also about the CD flow, I think it should be included.